internal/handler/http.go

```diff
@@ -141,6 +141,7 @@ func (sh *ServerHandler) ServeHTTP(response http.ResponseWriter, request *http.R
 	sh.handleRequest(sh.fixWrp(msg))
 
 	// return a 202
+	//TODO: do we need to add error handling because i'm getting this response even when event had not been placed on queue
```

Comment:
Depends how the code is setup...


for now a 202 is valid since the `server [has] received the HTTP request and intends to process it` (the meaning of 202) even tho it may fail in the down stream logic

Reply:
worth to make a note and come back